src/store/projects/project-create-actions.ts

```diff
@@ -63,3 +67,19 @@ export const createProject = (project: Partial<ProjectResource>) =>
             return undefined;
         }
     };
+
+export const addPropertyToCreateProjectForm = (data: ResourcePropertiesFormData) =>
+    (dispatch: Dispatch, getState: () => RootState, services: ServiceRepository) => {
+        const selector = formValueSelector(PROJECT_CREATE_FORM_NAME);
+        const properties = selector(getState(), 'properties') || {};
+        properties[data.key] = data.value;
+        dispatch(change(PROJECT_CREATE_FORM_NAME, 'properties', {...properties } ));
```

Comment:
It would be better to create copy on extracting properties from state (line 74) than here